src/website/app/Markdown.js

```diff
@@ -211,6 +199,28 @@ export default function Markdown({ children, scope, ...restProps }: Props) {
       },
       table({ children }) {
         return <MarkdownTable>{children}</MarkdownTable>;
+      },
+      li({ children }) {
+        const newChildren = children.map(child => {
```

Comment:
This can be simplified a bit to not rely on the index so much.  Consider the following...

```js
      li({ children }) {
        const newChildren = children.map((child, index) => {
          if (typeof child !== 'string') {
            return child;
          }

          const labelDelimiter = ' ~ ';
          const [preLabelText, labelText] = child.split(labelDelimiter);

          if (!labelText) {
            return child;
          }

          const labelVariant = labelText === 'done' ? 'success' : 'regular';

          return [
            preLabelText,
            <Label key={index} variant={labelVariant}>
              {labelText}
            </Label>
          ];
        });

        return <ListItem>{newChildren}</ListItem>;
      }
```

Reply:
i like the .trim() calls in there to be flexible with whitespace

Reply:
String.split accepts a regex, so you still can be.

Reply:
oh duh. nice